evansmusomi/react-native-102
# Adds Animations to Finding Places

*What this does*
- Adds a find places button with an exit animation
- Adds an animation when loading the list of added places

*How to verify*
- Share some places via the Share Place screen
- Navigate to the Find Place screen
- Press the "Find Places" button at the center
- The button should expand in size as it fades away
- Also, the list of added places should fade in to view
